2022-04-11 18:03:47,234 ********Appium Server Starting********
2022-04-11 18:03:49,571 Could not connect to port 4723 on host 127.0.0.1
2022-04-11 18:03:49,571 Could not get IP address for host: 127.0.0.1
2022-04-11 18:03:51,670 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AAE5D2B580>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /wd/hub/session
2022-04-11 18:03:53,710 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AAE5D2BA00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /wd/hub/session
2022-04-11 18:03:55,747 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AAE5D2BB50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /wd/hub/session
2022-04-11 18:04:02,844 ********Appium Server Starting********
2022-04-11 18:04:04,080 Could not connect to port 4723 on host 127.0.0.1
2022-04-11 18:04:04,081 Could not get IP address for host: 127.0.0.1
2022-04-11 18:04:06,114 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AAE5D6B490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /wd/hub/session
2022-04-11 18:04:08,143 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AAE5D6B580>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /wd/hub/session
2022-04-11 18:04:10,195 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AAE5D6B6D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /wd/hub/session
